bookwyrm/views/books/books.py

```diff
@@ -46,6 +50,9 @@ def get(self, request, book_id, user_statuses=False):
         if not book or not book.parent_work:
             raise Http404()
 
+        if r := maybe_redirect_local_path(request, book):
```

Comment:
Investigated more and this is falsely redirecting if the user statuses are set. Need to only redirect if user_statuses is false.